src/user/campaignList/CampaignList.tsx

```diff
@@ -83,7 +83,9 @@ export function CampaignList({ advertiserCampaigns, fromDate }: Props) {
                 <Tooltip title={`Edit ${r.name}`}>
                   <IconButton
                     onClick={() =>
-                      history.push(`/user/main/adsmanager/advanced/${r.id}`)
+                      history.push(
+                        `/user/main/adsmanager/advanced/${r.id}/settings`
+                      )
```

Comment:
nit: not requested as part of this PR, but going forward could we introduce some simple functions to build all urls like we have in ads-admin?  

Will make it easier to overhaul the ur structure in due course.